assets/src/edit-story/components/form/color/applyOpacityChange.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function applyOpacityChange(pattern, newOpacity) {
+  if (!pattern) {
+    return null;
+  }
+
+  const { type } = pattern;
+
+  // Return the same color, just with opacity applied
+  if (!type || type === 'solid') {
+    const {
+      color: { r, g, b },
+    } = pattern;
+    return { color: { r, g, b, a: newOpacity } };
+  }
+
+  // TODO: Return gradient with opacity applied
```

Comment:
LG. But to clarify, I confirmed this with Paul, and he definitely wants a persisted `a` multiplier for gradient as well. I.e. no need to immediately multiply it into color steps.

Reply:
Will do that in #497.

Reply:
Done in 5feec60e.